vng_api_common/audittrails/models.py

```diff
@@ -49,6 +49,10 @@ class AuditTrail(models.Model):
         auto_now=True,
         help_text=_("De datum waarop de handeling is gedaan")
     )
+    resource_weergave = models.URLField(
```

Comment:
`URLField` is niet correct hiervoor :-) `CharField` lijkt me correcter

Reply:
Hoe dit uberhaupt werkte met URLField is me een raadsel haha